src/index.css

```diff
@@ -1,13 +1,18 @@
+@import url("https://chunksui.netlify.app/css/style.css");
```

Comment:
I think linking the css in `index.html` is better for performance since we can use features like `preconnect` for stylesheets